python/tvm/relay/frontend/tflite.py

```diff
@@ -2726,7 +2726,13 @@ def convert_quantize(self, op):
         assert len(input_tensors) == 1, "input tensors length should be 1"
         input_tensor = input_tensors[0]
         input_tensor_type_str = self.get_tensor_type_str(input_tensor.tensor.Type())
-        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        if self.has_expr(input_tensor.tensor_idx):
+            in_expr = self.get_expr(input_tensor.tensor_idx)
+        else:
+            in_value = self.get_tensor_value(input_tensor)
```

Comment:
Is it possible to share the model? I am wondering if parsing of constants is the problem here. We can think of this case just like conv where weights are converted to relay consts and we add a var for them.

Reply:
Deepspeech itself is here https://github.com/mozilla/DeepSpeech/releases/tag/v0.7.4
However the model which was used was a quantised version fro internal model zoo. I am not sure that I can share it fully